src/Libraries/RevitNodes/Filter/OverrideGraphicSettings.cs

```diff
@@ -50,18 +44,24 @@ internal OverrideGraphicSettings(Autodesk.Revit.DB.OverrideGraphicSettings inter
         /// <param name="projectionFillPattern">Projection fill pattern</param>
         /// <param name="cutLinePattern">Cut line pattern</param>
         /// <param name="projectionLinePattern">Projection line pattern</param>
-        /// <returns>OverrideGraphicSettings</returns>
+        /// <param name="transparency">Transparency as integer between 1-100.</param>
+        /// <param name="detailLevel">Detail Level setting, ex: Coarse, Fine etc.</param>
+        /// <param name="halftone">Halftone. True = halftone.</param>
+        /// <returns name="overrides">Override Graphic Settings</returns>
         public static OverrideGraphicSettings ByProperties(
```

Comment:
I agree its a good idea to add Fill Pattern Element to Dynamo. That will make it easier for people. 
Now, I don't agree with getting rid of `null` input. In this particular case I think it makes sense because it actually has a use and is a valid input. It resets the override to "by category". Also, a null used in this node will never escape into the wider context of the definition so I would not worry about it corrupting the graph.